lib/upnp/upnp.go

```diff
@@ -168,7 +168,7 @@ USER-AGENT: syncthing/1.0
 	_, err = socket.WriteTo(search, ssdp)
 	if err != nil {
 		if e, ok := err.(net.Error); !ok || !e.Timeout() {
-			l.Infoln(err)
+			l.Debugln(err)
 		}
```

Comment:
You should also tweak SetTCPOptions to not fail on first failure.

Reply:
I don't really know what you want here.